client/src/components/ReviewTile.js

```diff
@@ -1,23 +1,92 @@
-import React from "react";
+import React, { useState } from "react";
+import EditReviewForm from "./EditReviewForm";
+import ErrorList from "./layout/ErrorList";
 
-const ReviewTile = ({ rating, content, onDelete, id, currentUser, userId, userName }) => {
+const ReviewTile = ({ rating, content, onDelete, id, currentUser, userId, memeId, setMeme, meme }) => {
+  const [editForm, setEditForm] = useState(false);
+  const [errors, setErrors] = useState([]);
+
+  const handleEditButton = (event) => {
+    event.preventDefault();
+    if (editForm) {
+      setEditForm(false);
+    } else {
+      setEditForm(
+        <EditReviewForm
+          rating={rating}
+          content={content}
+          id={id}
+          setEditForm={setEditForm}
+          editReview={editReview}
+        />
```

Comment:
wooh woh woh

bad. We never want to add this component to state. Only normal data types!! 

You also dont _need_ to, just change editForm to true, and then use an if statement to conditionally render that EditReviewForm based on if `editForm` is true.

I respect the bold move, but this is very very antipattern